ui/src/app/cohort-search/search-bar/search-bar.component.tsx

```diff
@@ -0,0 +1,300 @@
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+import {domainToTitle} from 'app/cohort-search/utils';
+import {ClrIcon} from 'app/components/icons';
+import {TextInput} from 'app/components/inputs';
+import {TooltipTrigger} from 'app/components/popups';
+import {Spinner} from 'app/components/spinners';
+import {cohortBuilderApi} from 'app/services/swagger-fetch-clients';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {highlightSearchTerm, reactStyles} from 'app/utils';
+import {triggerEvent} from 'app/utils/analytics';
+import {currentWorkspaceStore} from 'app/utils/navigation';
+import {Criteria, CriteriaType, DomainType} from 'generated/fetch';
+import {Key} from 'ts-key-enum';
+
+const styles = reactStyles({
+  searchContainer: {
+    background: colors.white,
+    width: '100%',
+    zIndex: 10,
+  },
+  searchBar: {
+    backgroundColor: colorWithWhiteness(colors.secondary, 0.8),
+    borderRadius: '5px',
+    height: '40px',
+    padding: '7px 14px',
+  },
+  searchInput: {
+    background: 'transparent',
+    border: 0,
+    height: '1rem',
+    marginLeft: '0.25rem',
+    outline: 'none',
+    padding: '0',
+    width: '85%',
+  },
+  dropdownMenu: {
+    position: 'absolute',
+    top: '100%',
+    left: 0,
+    marginTop: '.083333rem',
+    display: 'flex',
+    flexDirection: 'column',
+    background: colors.white,
+    padding: '.25rem 0',
+    border: '1px solid #ccc',
```

Comment:
Nit: you can use colors variable here. Use backticks, and then you can insert variables into the string with ${}